app/controllers/admin/druid_search_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Admin
+  # Controller for searching for work or collection by druid
+  class DruidSearchController < Admin::ApplicationController
+    def new
+      authorize!
+
+      @druid_search_form = DruidSearchForm.new
+    end
+
+    def search
+      authorize!
+
+      @druid_search_form = DruidSearchForm.new(druid_search_form_params)
+      if @druid_search_form.valid?
+        redirect_to_work_or_collection
+      else
+        render :new, status: :unprocessable_entity
+      end
+    end
+
+    private
+
+    def redirect_to_work_or_collection
+      path = if @druid_search_form.collection.present?
+               collection_path(druid: @druid_search_form.druid)
+             else
+               work_path(druid: @druid_search_form.druid)
+             end
```

Comment:
I'm guessing this lives in the controller instead of the `DruidSearchForm` because the latter does not have path helpers?

Reply:
Yeah, that feels like overloading the form. I don't think any of our other forms support similar.

Reply:
@justinlittman Agree, this makes more sense